site/_posts/2016-10-03-jekyll-3-3-0-released.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: news_item
+title: 'Jekyll 3.3 is here with better theme support, new URL filters, and tons more'
+date: 2016-10-03 17:08:38 -0700
+author: parkr
+version: 3.3.0
+categories: [release]
+---
+
+We have tons of new features for you in our latest release of Jekyll. Three
```

Comment:
That sounds fine to me! Though I might replace 'give a whirl' with just 'try'. But that's the tech writer in me talking. :)